tests/ctest_ut/windows_types_tests.c

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "ctest.h"
+
+CTEST_BEGIN_TEST_SUITE(WindowsTypesTests)
+
+/* ULONG */
+
+CTEST_FUNCTION(Assert_Are_Equal_ULONG_Succeeds)
+{
+    CTEST_ASSERT_ARE_EQUAL(ULONG, 1, 1, "test arg:%d", 42);
```

Comment:
>"test arg:%d", 42); [](start = 40, length = 19)

is this 42 relevant to the test?

Reply:
Nope. Just an arg to pass to the message formatter... Can remove it, but figured that it does not really hurt.

---
In reply to: [309940053](https://github.com/Azure/azure-ctest/pull/34#discussion_r309940053) [](ancestors = 309940053)